HTTP_200/templates/about.html

```diff
@@ -205,7 +205,7 @@ <h4><a href="http://jssaten.ac.in/Academics/EE/" target="_blank">Electrical Engi
             <h4><a href="http://jssaten.ac.in/Academics/ECE/" target="_blank">Electronics & Communication Engineering</a></h4>
           </div>
           <div class="col col-3">
-            <h4><a href="http://jssaten.ac.in/Academics/IC" target="_blank">Instrumentation and Control</a></h4>
+            <h4><a href="http://jssaten.ac.in/Academics/ice/Index.php" target="_blank">Instrumentation and Control</a></h4>
           </div>
         </div>
       </div>
```

Comment:
<img width="1440" alt="screen shot 2018-07-08 at 3 24 33 pm" src="https://user-images.githubusercontent.com/12206047/42418691-2a918458-82c3-11e8-88e5-9d71ff2298f5.png">
Please replace `My Inbox` from second column with `MBA` and add the link `National Digital Library of India` with hyperlink `https://ndl.iitkgp.ac.in/` below the `JSS Library` by shifting it up.

Please let me know if anything isn't clear to you.

Reply:
Got it sir! Updating this.